server/src/binary/handlers/consumer_groups/create_consumer_group_handler.rs

```diff
@@ -27,18 +28,35 @@ pub async fn handle(
                 command.group_id,
                 &command.name,
             )
-            .await?;
+            .await
+            .with_error(|_| {
+                format!(
+                    "Failed to create consumer group for stream_id: {}, topic_id: {}, group_id: {:?}, session: {:?}",
+                    command.stream_id, command.topic_id, command.group_id, session
+                )
+            })?;
         let consumer_group = consumer_group.read().await;
         response = mapper::map_consumer_group(&consumer_group).await;
     }
+
     let system = system.read().await;
+    let stream_id = command.stream_id.clone();
```

Comment:
hmm, but without cloning, we cannot use these variables, because command borrowed [here](https://github.com/iggy-rs/iggy/pull/1309/files#diff-bbe52d554159054d1b50257f8c16ced389bd302fbbd36d73576d589429e4279cR51). Сorrect me if I'm wrong